app/helpers/complaints_helper.rb

```diff
@@ -35,4 +35,22 @@ def sort_date(date)
   def sort_status(status_label)
     STATUS_SORT_ORDER[status_label]
   end
+
+  def aria_sort(column)
+    current_sort_direction = current_sort_direction(column)
```

Comment:
[question] Is there a reason why you're overriding the function name with the same variable name here?